server/svix-server/tests/it/e2e_proxy.rs

```diff
@@ -17,21 +17,38 @@ async fn test_message_delivery_via_socks5() {
     use crate::utils::start_svix_server_with_cfg;
 
     let mut cfg = get_default_test_config();
-    cfg.proxy_config = Some(proxy_config());
+    cfg.proxy_config = Some(socks_proxy_config());
     let (client, _) = start_svix_server_with_cfg(&cfg).await;
-    run_socks5_test(&client).await;
+    run_proxy_test(&client).await;
 }
 
-fn proxy_config() -> ProxyConfig {
+fn socks_proxy_config() -> ProxyConfig {
     ProxyConfig {
         addr: ProxyAddr::new("socks5://localhost:1080").unwrap(),
     }
 }
 
-async fn run_socks5_test(client: &TestClient) {
+#[ignore] // requires an http proxy to be running
```

Comment:
Unlike last time, I didn't even try to add `tinyproxy` to our docker compose stack. I could find way too many images for it on Docker Hub, but none looked sufficiently trustworthy. I tested this with a locally-running proxy instead. Suggestions for a Docker image to use would be welcome, but I think it's not required for merging this.